src/QsCompiler/Tests.Compiler/Tests.Compiler.fsproj

```diff
@@ -175,8 +178,12 @@
     <PropertyGroup>
       <ExecutionTarget>$(MSBuildThisFileDirectory)..\TestTargets\Simulation\Example\bin\$(Configuration)\netcoreapp3.0\Example.dll</ExecutionTarget>
       <LibraryTarget>$(MSBuildThisFileDirectory)..\TestTargets\Libraries\Library1\bin\$(Configuration)\netcoreapp3.0\Library1.dll</LibraryTarget>
+      <LibraryTargetCopy>$(MSBuildThisFileDirectory)..\TestTargets\Libraries\Library1\bin\$(Configuration)\netcoreapp3.0\Library2.dll</LibraryTargetCopy>
     </PropertyGroup>
-    <WriteLinesToFile File="$(OutputPath)ReferenceTargets.txt" Lines="$(ExecutionTarget); $(LibraryTarget)" Overwrite="true" />
+    <Copy SourceFiles="$(LibraryTarget)" DestinationFiles="$(LibraryTargetCopy)"/>
```

Comment:
Ah, I see. I was going to suggest to actually manually make literally another library project and add it to the solution instead of copying it here.

Reply:
I was trying to avoid going to all the trouble when I only need a copy. :) Do you think copying the file isn't a good idea?

Reply:
Having a second library just seems cleaner to me, and not all that much trouble. I meant to make one for testing purposes, but so far there just wasn't a reason to.

Reply:
Oh, actually, that is a good point... I forgot Library1 contains public methods besides the internal ones in the access modifier tests. That would probably make the test fail even if internal redeclarations were supported correctly. I'll make a separate library.

Reply:
I decided to write a completely different test that more precisely checks that specializations with the same name are grouped correctly based on their assembly. (The old test was only checking that the name still existed in the syntax tree.) I added it to the linking tests. The extra library file is no longer needed. Please take another look. :)

Reply:
Looks good to me.